client/my-sites/site-settings/settings-newsletter/SubscribeModalOnCommentSetting.tsx

```diff
@@ -18,15 +19,16 @@ export const SubscribeModalOnCommentSetting = ( {
 
 	return (
 		<>
+			<SupportInfo
+				text={ translate( 'Ask your readers to subscribe after commenting.' ) }
```

Comment:
You could add `hasTranslation` check here to be able to merge earlier before translations are done:

https://github.com/Automattic/wp-calypso/blob/32ed6aaff1dc353fc06275bc2a57b974168fcb15/client/components/domains/use-my-domain/domain-input.jsx#L51